app/controllers/api/v1/vocabularies_controller.rb

```diff
@@ -0,0 +1,73 @@
+# frozen_string_literal: true
+
+###
+# @description: Place all the actions related to vocabularies
+###
+class Api::V1::VocabulariesController < ApplicationController
+  before_action :authorize_with_policy
+
+  include Pundit
```

Comment:
The `Pundit` module is already included in the base class, `ApplicationController`.

Reply:
Ok, my bad. I removed it from the rest of the controllers also.